perception/autoware_shape_estimation/lib/model/bounding_box.cpp

```diff
@@ -159,6 +166,28 @@ bool BoundingBoxShapeModel::fitLShape(
   pose_output.position.x = (intersection_x_1 + intersection_x_2) * 0.5;
   pose_output.position.y = (intersection_y_1 + intersection_y_2) * 0.5;
   pose_output.position.z = min_z + shape_output.dimensions.z * 0.5;
+  double cluster_angle_rad = std::atan2(pose_output.position.y, pose_output.position.x);
+
+  if (-FRONT_BACK_ANGLE_THRESHOLD <= cluster_angle_rad || 
+      cluster_angle_rad < FRONT_BACK_ANGLE_THRESHOLD) { // front
```

Comment:
I think this is a bug.  
Since `||` is used here, all the condition will go in this branch.
`&&` should be used.

Reply:
fixed at [f022e2b](https://github.com/autowarefoundation/autoware_universe/pull/11309/commits/f022e2b07497b9e0cae51ed23b8682b0754aa15e)